src/popup/router/pages/Intro.vue

```diff
@@ -138,6 +138,10 @@ export default {
         privateKey: seed,
       };
       await this.$store.dispatch('setLogin', { keypair });
+      this.$store.commit('addNotification', {
+        text: this.$t('pages.account.seedNotification', [this.$t('pages.account.backup')]),
+        path: '/securitySettings',
+      });
```

Comment:
Was it intended to show this only the first time, not on each opening as it was before?